scripts/tests/twister_blackbox/test_printouts.py

```diff
@@ -166,3 +175,41 @@ def test_tree(self, capfd, test_path, expected):
 
         assert expected in out
         assert str(sys_exit.value) == '0'
+
+    @pytest.mark.usefixtures("clear_log")
+    @pytest.mark.parametrize(
+        'test_path, test_platforms',
+        TESTDATA_4,
+        ids=['tests']
+    )
+
+    def test_timestamps(self, capfd, test_path, test_platforms):
+
+        args = ['-i', '-T', test_path, '--timestamps', '-v'] + \
+               [val for pair in zip(
+                   ['-p'] * len(test_platforms), test_platforms
+               ) for val in pair]
+
+        with mock.patch.object(sys, 'argv', [sys.argv[0]] + args), \
+            pytest.raises(SystemExit) as sys_exit:
+            self.loader.exec_module(self.twister_module)
+
+        info_regex = r'\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2},\d{3} - (?:INFO|DEBUG|ERROR)'
+
+        out, err = capfd.readouterr()
+        sys.stdout.write(out)
+        sys.stderr.write(err)
+
+        output = err.split('\n')
+
+        err_lines=[]
```

Comment:
Spaces around the equals sign

Reply:
fixed

Reply:
I've meant `err_lines = []` rather than `err_lines=[ ]`

Reply:
oh, ok... thx 😃